modules/uhttp/client/client.go

```diff
@@ -39,16 +40,18 @@ var (
 // Client wraps around a http client
 type Client struct {
 	*http.Client
+	info                auth.CreateAuthInfo
```

Comment:
If we save it in auth filter, we will not have access to it if Client is created independently (without using new) Few lines below in Do, we inject default filters for this edge case.